process-services/latest/config/index.md

```diff
@@ -115,7 +115,8 @@ cors.support.credentials=truecors.preflight.maxage=10
 
 |Property|Description|
 |--------|-----------|
-|cors.allowed.origins|Specifies the hosts allowed in cross origin requests. By default, the value is set to `*`, which permits clients hosted on any server to access the resources. Alternatively, you can specify a host, for example, `http://www.example.org:8080`, which will only allow requests from this host.<br><br>Multiple entries or wildcards are not allowed for this setting.<br><br>In general, it is recommended to restrict `{{allowedOrigins}}` to only allow origins within your organization to make requests.|
+|cors.allowed.origins|Specifies A list of origins for which cross-origin requests are permitted. A value specified may be a specific domain, e.g `https://domain32.com`, or the CORS defined special value `*` for all origins. For matched pre-flight and actual requests the Access-Control-Allow-Origin response header can be either be the matched domain value or to `*`. **Note:** The CORS spec does not allow `*` when `allowCredentials` is set to `true` and from version 5.3 the combination is rejected and `cors.allowed.origin.patterns` should be used instead.|
```

Comment:
Can we remove `and from version 5.3 the combination is rejected` and instead just say that "`cors.allowed.origin.patterns` should be used instead"

Reply:
fixed done